lab_1_classify_by_unigrams/start.py

```diff
@@ -3,6 +3,60 @@
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
 
+def tokenize(text):
+
+    text = text.lower()
```

Comment:
in start.py you have to show the work of your written code, how it works

Reply:
done